test/spec/trello_helper.rb

```diff
@@ -0,0 +1,42 @@
+class TrelloHelper
```

Comment:
I would rather see this called `TrelloIntegrationHelper` or something to make it more obvious it's used only in test.